app/templates/admin/add_tags.html

```diff
@@ -15,83 +19,56 @@ <h2 class="ui header">
+                        <input id="search-users" type="text" placeholder="Search tags or type…">
                         <i class="search icon"></i>
                     </div>
                 </div>
             </div>
 
             {# Use overflow-x: scroll so that mobile views don't freak out
              # when the table is too wide #}
-           <!--  {% for t in tags %} -->
-                <div style="overflow-x: scroll;">
-                <h3 class="ui header">
-                    Age Group
-                </h3>
-                    <table class="ui searchable sortable unstackable selectable celled table">
-                        <thead>
-                            <tr>
-                                <th class="sorted ascending">Tag Name</th>
-                            </tr>
-                        </thead>
-                        <tbody>
-                           <!--  {% if t.tag_type == "Service" %}
-                                <tr>
-                                    <td>{{ t.tag_name }}</td>
-                                </tr>
-                            {% endif %} -->
-                                <!-- <tr> <button onclick="window.location.href = '{{ url_for('admin.add_new_tag', tag_type=age_group) }}';" class="ui basic compact button"> Yes </button> </tr> -->
-                        </tbody>
-                    </table>
-                </div>
-          <!--   {% endfor %} -->
-                <div style="overflow-x: scroll;">
-                <h3 class="ui header">
-                    Service
-                </h3>
-                    <table class="ui searchable sortable unstackable selectable celled table">
-                        <thead>
-                            <tr>
-                                <th class="sorted ascending">Tag Name</th>
-                            </tr>
-                        </thead>
-                        <tbody>
-                           <!--  {% if t.tag_type == "Age Group" %}
-                                <tr>
-                                    <td>{{ t.tag_name }}</td>
-                                </tr>
-                            {% endif %} -->
-                                <!-- <tr> <button onclick="window.location.href = '{{ url_for('admin.add_new_tag', tag_type=service) }}';" class="ui basic compact button"> Yes </button> </tr> -->
-                        </tbody>
-                    </table>
-                </div>
+             {{ f.render_form(form)}}
+             <div style="overflow-x: scroll;">
+                <table class="ui searchable sortable unstackable selectable celled table">
+                    <thead>
+                        <tr>
+                            <th>Tag Name</th>
+                            <th>Tag Type</th>
+                        </tr>
+                    </thead>
+                    <tbody>
+                    {% for t in tags %}

```

Comment:
TODO: @rani, change add_fake_data to make tags of fixed tag types (not a random word) so the rendering is correct.

Reply:
The set of tags types is fixed, but the client has changed them a bit since our first convo. Can you change the list of tag types to include these: 
<img width="689" alt="screen shot 2018-04-06 at 3 05 33 pm" src="https://user-images.githubusercontent.com/9835059/38439554-0e5a2ff2-39ac-11e8-9461-5880837be819.png">

Also, a the best way to do some of this would maybe be to generalize and send the whole "tag_types" to iterate over, instead of hardcoding the set of tag types -- makes the code more robust.